tools/eslint-rules/rules/no-terminal-popups.ts

```diff
@@ -18,50 +18,73 @@ export const rule = ESLintUtils.RuleCreator(() => __filename)({
   },
   defaultOptions: [],
   create(context) {
+    const childProcessMethods = new Set([
+      'exec',
+      'execSync',
+      'execFile',
+      'execFileSync',
+      'spawn',
+      'spawnSync',
+    ]);
+
+    const checkWindowsHide = (fnName: string, args: any[]) => {
+      let optionsArg;
+
+      switch (fnName) {
+        case 'exec':
+        case 'execSync':
+          optionsArg = args[1]; // for exec command
+          break;
+        case 'execFile':
+        case 'execFileSync':
+        case 'spawn':
+        case 'spawnSync':
+          optionsArg = args[2]; // for execFile / spawn command
+          break;
+      }
+
+      if (optionsArg && optionsArg.type === 'ObjectExpression') {
+        const hideWindowsProperty = optionsArg.properties.find(
+          (prop: any) =>
+            prop.type === 'Property' &&
+            prop.key.type === 'Identifier' &&
+            prop.key.name === 'windowsHide' &&
+            prop.value.type === 'Literal' &&
+            prop.value.value === true,
+        );
+        return !!hideWindowsProperty;
+      }
+      return false;
+    };
+
     return {
       CallExpression(node) {
-        const { callee, arguments: args } = node;
-
+        const callee = node.callee;
         if (
           callee.type === 'Identifier' &&
-          [
-            'exec',
-            'execSync',
-            'execFile',
-            'execFileSync',
-            'spawn',
-            'spawnSync',
-          ].includes(callee.name)
+          childProcessMethods.has(callee.name)
         ) {
-          let optionsArg;
-
-          switch (callee.name) {
-            case 'exec': // exec(command[, options][, callback])
-            case 'execSync': // execSync(command[, options])
-              optionsArg = args[1];
-              break;
-
-            case 'execFile': // execFile(file[, args][, options][, callback])
-            case 'execFileSync': // execFileSync(file[, args][, options])
-            case 'spawn': // spawn(command[, args][, options])
-            case 'spawnSync': // spawnSync(command[, args][, options])
-              optionsArg = args[2];
-              break;
+          const includeWindowsHide = checkWindowsHide(
+            (node.callee as any).name,
```

Comment:
try `callee.name,` here to get rid of `any`

Reply:
does the job